core-ui/src/shared/ResourceForm/fields/RichEditorDataField/RichEditorSection.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react';
+import { Button } from 'fundamental-react';
+import { isNil } from 'lodash';
+import { useTranslation } from 'react-i18next';
+import * as Inputs from 'shared/ResourceForm/inputs';
+import { FormField } from '../../components/FormField';
+import { Editor } from 'shared/components/MonacoEditorESM/Editor';
+import { ResourceForm } from 'shared/ResourceForm/components/ResourceForm';
+import { Dropdown } from 'shared/components/Dropdown/Dropdown';
+import { getAvailableLanguages } from './languages';
+
+export function RichEditorSection({ item, onChange, onDelete, pushValue }) {
+  const { t } = useTranslation();
+  const { key, value, language } = item || {};
+
+  const languageDropdown = (
+    <Dropdown
+      disabled={!item}
+      compact
+      options={getAvailableLanguages().map(l => ({
+        key: l.id,
+        text: l.aliases?.[0] || l.id,
+      }))}
+      selectedKey={language || ''}
+      onSelect={(e, { key: language }) => {
+        e.stopPropagation(); // don't collapse the section
+        onChange({ language });
+      }}
+      placeholder={t('common.statuses.unknown')}
```

Comment:
Could we add a more descriptive placeholder like 'choose a language' etc? We have something like this for presets.

Reply:
1. Me and @wchrapka are on it.
2. Damn, I was thinking about this, but I didn't want to add too many changes in this PR. I'll do it here then, sure.
3. I'll add a better placeholder.